homeassistant/components/qwikswitch/sensor.py

```diff
@@ -34,8 +34,10 @@ def __init__(self, sensor):
         sensor_type = sensor["type"]
 
         self._decode, self.unit = SENSORS[sensor_type]
-        if isinstance(self.unit, type):
-            self.unit = f"{sensor_type}:{self.channel}"
```

Comment:
the problem is that there is no way to test it in this case, since all the sensor types that have types are bool. If you look at the qwikswitch.py file in the library, they are all either of type bool, which would create a binary sensor, or the are strings

Reply:
Ok, let me have a look at how qwikcord is used...
Its one of those I added for someone else, but don't have the device myself

Reply:
Ah, I see whta you mean, I think here the intention was really just to protect if not a string, so this is ok

Reply:
thanks